versions/DEVELOPMENT-SNAPSHOT-2017-02-20-a.yaml

```diff
@@ -0,0 +1,5 @@
+binaries:
+  osx: https://swift.org/builds/development/xcode/swift-DEVELOPMENT-SNAPSHOT-2017-02-20-a/swift-DEVELOPMENT-SNAPSHOT-2017-02-20-a-osx.pkg
+  ubuntu14.04: https://swift.org/builds/development/ubuntu1404/swift-DEVELOPMENT-SNAPSHOT-2017-02-20-a/swift-DEVELOPMENT-SNAPSHOT-2017-02-20-a-ubuntu14.04.tar.gz
+  ubuntu16.04: https://swift.org/builds/development/ubuntu1604/swift-DEVELOPMENT-SNAPSHOT-2017-02-20-a/swift-DEVELOPMENT-SNAPSHOT-2017-02-20-a-ubuntu16.04.tar.gz
+  ubuntu16.10: https://swift.org/builds/development/ubuntu1404/swift-DEVELOPMENT-SNAPSHOT-2017-02-20-a/swift-DEVELOPMENT-SNAPSHOT-2017-02-20-a-ubuntu14.04.tar.gz
```

Comment:
Sorry, my bad! Corrected in https://github.com/kylef/swiftenv-api/pull/26/commits/ec34014b48a26f32bc189f7978810eed798b341e